app/plans/index.njk

```diff
@@ -41,7 +41,7 @@
 
                 {{ govukButton({
                     text: 'Continue',
-                    href: "/individual-id/" + individualId + "/edit-plan/"
+                    href: "/individual-id/" + individualId + "/edit-plan/" + currentPlan.planId
```

Comment:
Can you just double check my reading of this is correct.
I've added the planId to the url, because otherwise this url doesn't point to the existing draft plan?